admin_manual/maintenance/migrating.rst

```diff
@@ -62,3 +62,138 @@ When a user tries a URL that is not whitelisted the following error appears:
    :scale: 75%
    :alt: Error message when URL is not whitelisted
 
+=================
+Example migration
+=================
+
+.. note:: For this example to work, you need this on both servers:
+
+* Ubuntu 16.04
+* SSH
+* PermitRootLogin set to "yes"
+* network
+
+optional:
+
+* Domain Name
+* modules: smb-client nfs-common rpcbin
+
+
+Install SSH::
+
+   apt install openssh-server openssh-client -y
+
+Edit SSH-Config::
+
+   nano /etc/ssh/sshd_config
+
+Change PermitRootLogin to yes::
+
+   PermitRootLogin yes
+
+=========
+Migration
+=========
+
+1. Install ownCloud on new server
+
+If you have NAS, then you will need the ```smb-client nfs-common rpcbin``` modules.
+
+2. Put original server in maintenance mode:
+
+Go in owncloud dir::
+
+      cd /var/www/owncloud/
+
+
+Switch to maintenance mode::
+
+      sudo -u www-data php occ maintenance:mode --on
+
+
+wait for 6-7 min and stop apache2::
+
+   service apache2 stop
+
+3. Export and Import the database
+
+Go in owncloud dir::
+
+      cd /var/www/owncloud/
+
+Backup the database:
+
+SYNOPSIS::
+
+   mysqldump --single-transaction -h [server] -u [username] -p[password] [db_name] > owncloud-dbbackup_`date +"%Y%m%d"`.bak
+
+Example::
+
+   mysqldump --single-transaction -h localhost -u admin -ppassword owncloud > owncloud-dbbackup.bak
+
+.. note:: You can find the values for the mysqldump command in your config.php at your owncloud directory.
+[server]= dbhost, [username]= dbuser, [password]= dbpassword, and [db_name]= dbname.
+
+.. note:: For InnoDB tables only:
+The --single-transaction flag will start a transaction before running. Rather than lock the entire database, this will let mysqldump read the database in the current state at the time of the transaction, making for a consistent data dump.
+
+For Mixed MyISAM / InnoDB tables:
+Either dumping your MyISAM tables separately from InnoDB tables or use --lock-tables instead of --single-transaction to guarantee the database is in a consistent state when using mysqldump.
+
+
+Export on original server::
+
+   rsync -Aaxt owncloud-dbbackup.bak root@new_server_address:/var/www/owncloud 
+
+Import on new server::
+
+   mysql -h localhost -u admin -ppassword owncloud < owncloud-dbbackup.bak
+
+4. Copy data, config to new server::
+
+      rsync -Aavxt config data root@new_server_address:/var/www/owncloud 
+
+.. warning:: If you want to move your datadirectory to another location on the target server, it is advised to do this as a second step. Please see the datadirectory migration document :ref:`datadir_move_label` for more details.
```

Comment:
Point 4.? 

4. Copy data, config to new server::

      rsync -Aavxt config data root@new_server_address:/var/www/owncloud 

.. warning:: If you want to move your datadirectory to another location on the target server, it is advised to do this as a second step. Please see the datadirectory migration document :ref:`datadir_move_label` for more details.

Or do you mean in the other doc?

@PVince81 I migrated a fresh installed ownCloud server, and wrote my steps down as an example, then edited the example with the suggestions, I know that my example worked, not sure about the stuff people suggested though, because I have not tested it. But I think it works, based on the people who made the suggestions